nav2_bringup/launch/localization_launch.py

```diff
@@ -51,16 +51,6 @@ def generate_launch_description():
     remappings = [('/tf', 'tf'),
                   ('/tf_static', 'tf_static')]
 
-    # Create our own temporary YAML files that include substitutions
-    param_substitutions = {
-        'yaml_filename': map_yaml_file}
-
-    configured_params = RewrittenYaml(
```

Comment:
Still need this for injecting namespaces

Reply:
@SteveMacenski  take a look at https://github.com/ros2/rclcpp/pull/1839, I think with this you could remove the RewrittenYaml (for injecting a namespace at least)

Reply:
I am not then required to leave it then?, also it is worth mentioning that the autostart parameter for the nav2_container still uses the RewrittenYaml.

Reply:
This would need some additional work, it's not for this PR imo. So you still need it for now.